### ISSUE TITLE
Integer validation error

### ISSUE THREAD

**Issue by niklv** [ajv-validator/ajv#490]
I use ajv@5.1.0.

**Ajv options object**
```javascript
{}
```

**JSON Schema**
```json
{
  "type": "integer",
  "minimum": 0
}
```

**Sample data**

```json
"abc"
```

In node REPL:

```javascript
> const Ajv = require('ajv');
undefined
> const validator = (new Ajv({})).compile({type: 'integer', minimum: 0});
undefined
> validator(1);
true
> validator('1');
true
> validator('abc');
true
```

**What results did you expect?**
Expect to validate integer.

**Comment by epoberezkin** [comment]
@Niklv thanks for reporting.
That's the consequence of #485.
Version 5.0.1 is ok

**Comment by epoberezkin** [comment]
fixed in 5.1.1

**Comment by prasadrails** [comment]
Hi @epoberezkin ,
I also used same data which is delcared in this ticket.
> validator(1.8);
false
> validator(1.000000000000000000000008);
true

The root cause is 
> Number.isInteger(1.000000000000000000000008);
true

What results did you expect?
Expect to validate integer.

Thanks in advance,
Durga Prasad.
